capitoli/Quinto_Passo_Primi_Programmi.rst

```diff
@@ -338,14 +341,13 @@ Per ogni esercizio assegnati un punto se riesci a trovare l’errore e un altro
 ::
 
        # Es. 2  	
-	Input di numeri e stringhe
-
-	Primonumero = int(input ("Scrivi il primo numero:  "))
-	Secondonumero = int(input ("Scrivi il secondo numero: "))
-	Nome = input ("Scrivi il tuo nome:  ")
-	Cognome = input ("Scrivi il tuo cognome:  ")
-	Print nome , cognome, "primonumero", "per", secondonumero, "uguale", 
-	primonumero*secondonumeeo
+       #Input di numeri e stringhe
+
+       Primonumero = int(input ("Scrivi il primo numero:  "))
+       Secondonumero = int(input ("Scrivi il secondo numero: "))
+       Nome = input ("Scrivi il tuo nome:  ")
+       Cognome = input ("Scrivi il tuo cognome:  ")
+       Print (nome, cognome, primonumero, "per", secondonumero, "uguale", primonumero*secondonumero)
```

Comment:
```suggestion
       print(nome, cognome, primonumero, "per", secondonumero, "uguale", primonumero*secondonumero)
```
Print minuscola, e direi di mettere le parentesi senza spazio dopo il nome della funzione.

Reply:
ma questo e'  "Caccia all'errore" e Print in maiuscolo l'avevamo lasciato perche' lo correggessero.